syspurpose/po/zanata.xml

```diff
@@ -1,18 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <config xmlns="http://zanata.org/namespace/config/">
-    <url>https://vendors.zanata.redhat.com/</url>
+    <url>https://fedora.zanata.org/</url>
     <project>syspurpose</project>
+    <project-version>RHEL-7.8</project-version>
```

Comment:
Let's leave the version out, makes it easier to update the translations via the cli (as we can specify which version at push and pull time as well)

Reply:
Done.